src/apps/util/config.clj

```diff
@@ -123,6 +123,16 @@
   [props config-valid configs]
   "apps.tools.private.memory-limit" (* 16 1024 1024 1024)) ;; 16GB
 
+(cc/defprop-optint default-cpu-limit
+  "The CPU limit, in cores (matching the database's data type), to be used as the default resource limit when one is not set in a tool definition."
+  [props config-valid configs]
+  "apps.tools.default.cpu-limit" 4)
+
+(cc/defprop-optint default-memory-limit
+  "The memory limit, in bytes, to be used as the default resource limit when one is not set in a tool definition."
+  [props config-valid configs]
+  "apps.tools.default.memory-limit" (* 16 1024 1024 1024)) ;; 16GB
```

Comment:
This one is higher than the hardcoded one in app-exposer, which is 8Gi instead, in order to match the limit when adding private tools that's further up in this file. Plus, 8Gi is pretty small for a limit anyway.

Reply:
👍